incubator/nfs-pv-provider/templates/nfs-server-pvc.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: "{{ .Values.server.persistentVolume.name }}"
+  labels:
+    app: {{ template "nfs-pv-provider.name" . }}
+    chart: {{ template "nfs-pv-provider.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    demo: "{{ .Values.server.persistentVolume.demo_label }}"
```

Comment:
What's the demo label for? And again, camel case only please for values.

Reply:
Removed it in 2aac6782ce573236f3cf68b3fed59e3e97f17727